app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+def create_name_file(name_file, input_line):
```

Comment:
`input_line` can be a confusing name for this argument (it can contain multiple lines of the content)
`name_file` -> `file_name`
`create_name_file` also a bit strange name :)

Reply:
heh) okay, I've fixed) thx